src/lib/swapper/swappers/LifiSwapper/utils/getIntermediaryTransactionOutputs/getIntermediaryTransactionOutputs.test.ts

```diff
@@ -26,9 +26,9 @@ describe('getIntermediaryTransactionOutputs', () => {
     expect(usdcOnEthereum).not.toBe(undefined)
     expect(usdcOnOptimism).not.toBe(undefined)
 
-    const expectation: IntermediaryTransactionOutput[] = [
-      { asset: usdcOnEthereum, buyAmountCryptoBaseUnit: '120527596' },
-      { asset: usdcOnOptimism, buyAmountCryptoBaseUnit: '120328249' },
+    const expectation: AmountDisplayMeta[] = [
+      { ...usdcOnEthereum, amountCryptoBaseUnit: '120527596' },
+      { ...usdcOnOptimism, amountCryptoBaseUnit: '120328249' },
```

Comment:
Ditto don't spread and would prefer to keep the partial asset as a separate property